### PR TITLE
Update account index empty, gateway index empty pages + buttons

### PR DESCRIPTION
**Pull request by vicmgs**
- Update account index empty page colors
- Update gateway index empty page colors
- Update reminder popups colors
- Refactor StyledButton implementation of colors
- Fix block explorer page breaking when node not connected

### CODE REVIEW

File: app/components/account/AccountNameModal.jsx [hunk 1]
```diff
@@ -85,13 +86,15 @@ class AccountNameModal extends Component {
             <StyledButton
               color="primary"
               onClick={this.closeModal}
+              colorHex={themeDefaults.palette.primary.main}
```
Comment (allenan) [inline app/components/account/AccountNameModal.jsx]: it seems a little non-optimal to need to require themeDefaults in order to use the StyledButton. why doesn't `color="primary"` work?

**Comment by allenan** [comment]
can you post some screenshots of the empty states?

**Comment by vicmgs** [comment]
screen shot 2018-12-19 at 12 35 02 pm
screen shot 2018-12-19 at 12 35 07 pm
screen shot 2018-12-19 at 12 36 16 pm

File: app/components/account/AccountNameModal.jsx (hunk 1 above)
Comment (vicmgs) [reply app/components/account/AccountNameModal.jsx]: fixed it so that only color passed to StyledButton is the hex

Comment (allenan) [reply app/components/account/AccountNameModal.jsx]: but why should you have to import `themeDefaults` into a component that's only concerned with a button? it seems like kind of a poor interface unless i'm missing something. ideally you'd just pass in `color="primary"` and the button figured out how to get that out of the theme

**Comment by allenan** [comment]
empty states look good

File: app/components/account/AccountNameModal.jsx (hunk 1 above)
Comment (vicmgs) [reply app/components/account/AccountNameModal.jsx]: Ok, updated. See styledButton file

**Pull request merged**